infrastructure/instances.tf

```diff
@@ -285,17 +265,15 @@ output "nomad_client_ip" {
   value = "${aws_instance.nomad_client_1.public_ip}"
 }
 
-variable "database_password" {}
-
 resource "aws_db_instance" "postgres_db" {
   identifier = "data-refinery-${var.user}-${var.stage}"
   allocated_storage = 100
   storage_type = "gp2"
   engine = "postgres"
-  engine_version = "9.5.4"
+  engine_version = "9.6.6"
   instance_class = "db.t2.micro"
-  name = "data_refinery_${var.user}_${var.stage}"
-  username = "data_refinery_user"
+  name = "data_refinery"
+  username = "${var.database_user}"
   password = "${var.database_password}"
   db_subnet_group_name = "${aws_db_subnet_group.data_refinery.name}"
   skip_final_snapshot = true
```

Comment:
May want to be false in prod.

Reply:
I would agree with this, but there's a [terraform issue](https://github.com/hashicorp/terraform/issues/5417) where you can't use terraform to destroy RDS instances with this property set to true without doing a workaround.

We'll definitely want this eventually, but the longer we punt on changing it the less headaches we'll have for dev infrastructure.